db/migrate/20180210123458_create_events_gas.rb

```diff
@@ -0,0 +1,13 @@
+class CreateEventsGas < ActiveRecord::Migration[5.1]
+  def change
+    create_table :events_gas do |t|
```

Comment:
It is not immediately obvious that `events_gas` is related to google analytics imo. I think `ga_events` or maybe even 'google_analytics_events` would be more obvious and easier for new developers to the project to grok

Reply:
@KyleMacPherson it was a legitimate concern, thank you for raising it. 
I think @chao-xian explained really well the background for the naming 👍